build_sysext

```diff
@@ -134,11 +143,34 @@ trap cleanup EXIT
 ARCH=$(_get_sysext_arch "${FLAGS_board}")
 cleanup
 
+# If we need to handle pkginfo squashfs files, create mount points under
+# ${BUILD_DIR}/base-pkginfo, mount the squashfs images, and add the mount paths to
+# the list of lowerdirs.
+pkginfo_lowerdirs=""
+if [[ -n "${FLAGS_base_pkginfo}" ]] ; then
+  for entry in $(echo ${FLAGS_base_pkginfo} | sed 's/:/ /g'); do
+    ppath="$(readlink -f "${entry}")"
+    if [[ ! -f "${ppath}" ]] ; then
+      error "--base_pkginfo contains invalid entries."
+      error "Pkginfo file '${ppath}' does not exist."
+      die "Full --base_pkginfo: '${FLAGS_base_pkginfo}'"
+    fi
+
+    pfile="$(basename "${ppath}")"
+    pmdir="${BUILD_DIR}/base-pkginfo/${pfile}"
+    mkdir -p "${pmdir}"
+    mount -rt squashfs -o loop,nodev "${ppath}" "${pmdir}"
+    pkginfo_lowerdirs="${pkginfo_lowerdirs}:${pmdir}"
```

Comment:
Is order of lowerdirs important here? (It's about unusual right-to-left ordering of layers in overlayfs lowerdir option).

Reply:
Should be fine here, same reason as for https://github.com/flatcar/scripts/pull/1216#discussion_r1367026876.